doc-independent-build.yaml

```diff
@@ -15,22 +15,21 @@ install_apt_packages:
 - python3-dateutil
 - python3-wstool
 - python3-yaml
+- python3-sphinx
+- python3-sphinx-rtd-theme
 install_pip_packages:
 - catkin-sphinx
-- elementpath==2.1.3  # pinned because newer version needs python >= 3.6: https://github.com/ros-infrastructure/rep/issues/304
-- jinja2==2.11.3
-- markupsafe==1.1.1
-- sphinx==3.5.4
-- xmlschema==1.2.5
+- xmlschema
```

Comment:
xmlschema and catkin-sphinx are not packaged in Ubuntu Focal. xmlschema is part of Hirsuite and Impish and so will likely be available in 22.04 when the time comes.

Reply:
Do you want to put this into a comment in the code so it's findable in the future by people reading the code.